Makefile

```diff
@@ -25,11 +25,7 @@ clean:  ## clean all build, python, and testing files
 	find . -name '*.pyo' -exec rm -fr {} +
```

Comment:
Random comment: I may swap `Make` out for `Just`. I like it a bit more (really only because I ran run `just` from any sub-directory and it has better function management imo.

Reply:
https://github.com/casey/just

Reply:
nice!